src/components/seo.js

```diff
@@ -32,9 +32,13 @@ const SEO = ({ canonical, description,image, schemaMarkup, title,children }) =>
     title: title || defaultTitle,
     description: description || defaultDescription,
     image: `${siteUrl}${image || siteMetadataImage}`,
-    url: `${siteUrl}${pathname || ""}`,
+    url: `${siteUrl}${pathname.replace(".html", "")  || ""}`,
```

Comment:
Curiious as to when can. we have `.html` in the path? We shouldn't be having it on any page, is it only if we intentionally try to do so? Forced testing?